pennylane/hf/integrals.py

```diff
@@ -14,8 +14,8 @@
 """
 This module contains the functions needed for computing integrals over basis functions.
 """
+# pylint: disable=unbalanced-tuple-unpacking, too-many-arguments
```

Comment:
What is unbalanced tuple unpacking? I've never seen that before 🤔

Reply:
It happens when `a, b, c = params` while `params = [k, l, m]` is used. I can fix it by changing the the function that returns `params` to 
`return params[0], params[1], params[2]`
instead of 
`return params`
I preferred to disable the warning but not sure if it is the better option.

Reply:
got it! no need to change it, was just curious 🙂